apps/bond/components/BondingProducts/Bonding/BondingList/BondingList.jsx

```diff
@@ -76,9 +76,21 @@ const getColumns = (onClick, isActive, acc, depositoryAddress, hideEmptyProducts
       },
     },
     {
-      title: getTitle('LP Token', 'LP token address enabled by the Treasury'),
+      title: getTitle('Liquidity Pool', 'Liquidity Pool on target network'),
```

Comment:
Currently, the list inconsistently sometimes links to the pool and sometimes to the token itself. They may or may not be the same, two columns are needed.